models/release.go

```diff
@@ -233,40 +233,44 @@ func GetReleaseByID(id int64) (*Release, error) {
 	return rel, nil
 }
 
-// GetReleasesByRepoID returns a list of releases of repository.
-func GetReleasesByRepoID(repoID int64, page, pageSize int) (rels []*Release, err error) {
-	if page <= 0 {
-		page = 1
-	}
-	err = x.
-		Desc("created_unix").
-		Limit(pageSize, (page-1)*pageSize).
-		Find(&rels, Release{RepoID: repoID})
-	return rels, err
+// FindReleasesOptions describes the conditions to Find releases
+type FindReleasesOptions struct {
+	IncludeDrafts bool
+	TagNames      []string
 }
 
-// GetReleaseCountByRepoID returns the count of releases of repository
-func GetReleaseCountByRepoID(repoID int64, includeDrafts bool) (int64, error) {
+func (opts *FindReleasesOptions) toConds(repoID int64) builder.Cond {
```

Comment:
But repoID is still allways required field and that will not easy be understandable by function signature